src/EiffelEvents.RabbitMq.Client/RabbitMqEiffelClient.cs

```diff
@@ -74,7 +92,21 @@ public RabbitMqEiffelClient(RabbitMqConfig config, int connectionRetries = 10)
         }
 
         /// <inheritdoc/>
-        public string Subscribe<T>(string serviceIdentifier, Action<T, ulong> callback) where T : IEiffelEvent, new()
+        public Result<T> Publish<T>(T eiffelEvent) where T : IEiffelEvent, new()
```

Comment:
I suggest (for clarity) to override docstring to mention that it is by default `ON` based on `ValidationConfig` in `ClientConfig` if not providede, as it could be changed globally in the configuration, hence the default could be `OFF` if it was set in the configuration.
So user never assume it is always `ON` even when changed globally.